src/frontend/screens/AppPasscode/NewPasscode/PasscodeIntro.tsx

```diff
@@ -26,12 +28,12 @@ interface SplashProps {
 }
 
 export const PasscodeIntro = ({ incrementState }: SplashProps) => {
-  const { setPassIsSet } = React.useContext(SecurityContext);
+  const [authState, setAuthState] = React.useContext(SecurityContext);
   const { formatMessage: t } = useIntl();
   const { navigate } = useNavigation();
 
   function tempStateSet() {
-    setPassIsSet(prev => !prev);
+    setAuthState({ type: "setPasscode", newPasscode: "12345" });
     navigate("Security");
   }
```

Comment:
couple of notes:

- would lean towards removing the navigation call here because with the screen transition, it feels broken/abrupt
- nit: if we keep the function here (which wouldn't be necessary if the first point is taken), can we rename it to something we wouldn't have to change later on?

Reply:
I am working on this part during this sprint, so this will be completely changed. This is just placeholder code!